submissions/akinbobolaFaith/css/index.css

```diff
@@ -0,0 +1,290 @@
+body {
+    margin: 0 -52px 0 0;
+    font-family: 'Lato' , sans-serif;
+    font-size: 16px;
+}
+
+* {
+    box-sizing: border-box;
+}
+
+nav a[href*="*"] {
+    font-size: 31px;
+    font-weight: bold;
+    color: white;
+}
+
+nav ul,
+.related-products ul,
+footer ul {
+    list-style-type: none;
+    margin: 0;
+    padding: 0;
+}
+
+a {
```

Comment:
@faithakinbobola you can group `a`, `a:link`, `a:hover`

Reply:
i have grouped a , a:link , a:hover